### DISCUSSION TITLE
Welcome to Ochi Discussions!

### DISCUSSION THREAD

**Discussion by glaslos** [honeynet/ochi#35] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ayushrai206** [comment]
Hello everyone, I am Ayushi, I am a past outreachy intern and I want to contribute to this project as a part of my GSoC proposal, can someone please guide me how to proceed with this?

**Comment by glaslos** [comment]
Hi Ayushi, I didn't plan on mentoring for Ochi this year. Any specific ideas or what you would like to contribute?

**Comment by ayushrai206** [comment]
Hey, Thank you so much for replying, I found this repository in the list of project ideas of The HoneyNet Project and among the list of ideas this one below matched my skillset the most so I decided to contribute to it-
#7 - Ochi – Honeypot Live Event Feed UI
Mentor: Lukas Rist

Project type: Improving existing tool

Project URL: https://github.com/glaslos/ochi

Project hours: 350

Ochi is a web interface for live Honeypot events to correlate, filter and annotate. In this project we would like to add the ability to filter the events using a version of the Wireshark query syntax as a DSL. Queries should be persistent and the backend should calculate metrics how often a query is matched.

Furthermore the backend should support perma-links to events so they can be shared.

Primary skills required are HTML/CSS/JavaScript. Secondary would be Golang but this can also be covered by the mentor of the project.

Do you suggest me back out and pick up any other idea or should I start with solving some github issues in the repository?
